pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeConsumptionRateManager.java

```diff
@@ -69,7 +73,8 @@ public void enableThrottling() {
     _isThrottlingAllowed = true;
   }
 
-  public ConsumptionRateLimiter createRateLimiter(StreamConfig streamConfig, String tableName) {
+  public ConsumptionRateLimiter createRateLimiter(StreamConfig streamConfig, String tableName,
```

Comment:
Can we overload this method and then call createRateLimiter(streamConfig, tableName, null, null) from the createRateLimiter(streamConfig, tabelName)? that reduces the touch points of the change as old invocations remain the same

Reply:
I didn't create one because the new overloaded method will be only used in test. That being said, your point is valid and we should treat test code as production code. Just updated the code.